lib/features/digital_guide_view/data/readme.md

```diff
@@ -0,0 +1,14 @@
+# Requirements
+* All HTTP requests must include the authorization token (DIGITAL_GUIDE_URL)
+
+# Tips
+* To obtain the authorization token, username, or password for API login, please contact the development team
+* The complete list of endpoints is available after logging under https://przewodnik.pwr.edu.pl/swagger/
+
+# Used endpoints and providers
+1) Building data
+  * DIGITAL_GUIDE_URL/buildings/{id}
+  * Used via: getDigitalGuideDataProvider({id})
```

Comment:
imo not needed here

Reply:
I'm not saying the repo name is the best thing to place here, but we need to have some info where we're using this specific endpoint in the code. But we could make up some "subfeatures" names for this instead? But imo the repo name is pretty cool too. Can stay for me